src/index.ts

```diff
@@ -47,15 +49,6 @@ interface ISbFlatMapped {
 	data: any
 }
 
-export interface ISbResponseData {
-	link_uuids: string[]
-	links: string[]
-	rel_uuids: string[]
-	rels: any
-	story: ISbStoryData
-	stories: Array<ISbStoryData>
-}
-
```

Comment:
Not actually deleted - just moved to the `interfaces.ts` file alongside other such interfaces.